system-test/storage.ts

```diff
@@ -2810,6 +2810,27 @@ describe('storage', () => {
       await Promise.all([file.delete, copiedFile.delete()]);
     });
 
+    it('should copy an existing file and overwrite metadata', async () => {
+      const opts = {
+        destination: 'CloudLogo',
+        metadata: {
+          metadata: {
+            originalProperty: 'true',
+          },
+        },
+      };
+      const [file] = await bucket.upload(FILES.logo.path, opts);
+      const copyOpts = {metadata: {newProperty: 'true'}};
```

Comment:
I'm a little confused about how this interface works. Here you have `newProperty`, in the PR description you have `thisIsACopy`, and above you have `originalProperty`. I see above you specify this field as type `Metadata`, which is in @google-cloud/common as `any`. How does the code know what to do with this property? Which one of these should it be specified as? Can we improve documentation for this?

Reply:
These are arbitrary property names and values. `metadata` is part of a GCS Object that can be used for user-defined key/values.